### PR TITLE
Refactor aps code

### PR DESCRIPTION
**Pull request by David-Gil**
DRY and document some functions used throughout `aps`.

### CODE REVIEW

File: aps/lib/aps.ex [hunk 1]
```diff
@@ -1,5 +1,86 @@
...
+    Application.get_env(:pummpcomm, :cgm)
+  end
+
+  @doc """
+  Returns true if the input is a sensor_glucose_value.
+  Otherwise, it returns false.
+
+  Used for sgv filtering with Enum.filter\1.
+  """
+  def filter_sgv({:sensor_glucose_value, _}), do: true
```
Comment (tmecklem) [inline aps/lib/aps.ex]: Filter and maps functions for sensor glucose feel out of place in a top level module like this. I'd prefer this live in a blood glucose specific module

File: aps/lib/aps.ex [hunk 2]
```diff
@@ -1,5 +1,86 @@
...
+  """
+  def cgm do
+    Application.get_env(:pummpcomm, :cgm)
+  end
+
+  @doc """
+  Returns true if the input is a sensor_glucose_value.
+  Otherwise, it returns false.
+
+  Used for sgv filtering with Enum.filter\1.
```
Comment (tmecklem) [inline aps/lib/aps.ex]: I think this should be a forward slash between the function name and arity

File: aps/lib/aps.ex (hunk 1 above)
Comment (David-Gil) [reply aps/lib/aps.ex]: Ok, I moved them to InfinityAPS.Glucose.Monitor.

File: aps/lib/aps.ex (hunk 2 above)
Comment (David-Gil) [reply aps/lib/aps.ex]: Indeed!!

**Pull request merged**